engine/src/test/scala/cromwell/ArrayAndMapIndexingWorkflowSpec.scala

```diff
@@ -10,10 +10,10 @@ import scala.language.postfixOps
 
 class ArrayAndMapIndexingWorkflowSpec extends CromwellTestkitSpec {
   "A workflow with array/map indexes in expressions" should {
-    "run locally" ignore {
+    "run locally" in {
       runWdlAndAssertOutputs(
         sampleWdl = SampleWdl.ArrayAndMapIndexingWdl,
-        eventFilter = EventFilter.info(pattern = s"starting calls: test.echo_int, test.echo_str", occurrences = 1),
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
```

Comment:
That assertion was in place to make sure the "find runnable calls" logic was finding the expected calls (in the very early days of Cromwell it wasn't a given that that logic was working correctly 😄 ).  Hopefully there's a more truly "unit" test for that functionality now and tests like this only need to concern themselves with asserting expected status and outputs.